### ISSUE TITLE
CVE in logstash jackson dependency

### ISSUE THREAD

**Issue by lread** [pyr/unilog#32]
Heya, thought I should let you know there is another jackson CVE.

If I install nvd-clojure:
```
$ clojure -Ttools install nvd-clojure/nvd-clojure '{:mvn/version "RELEASE"}' :as nvd
```
And then run it from a cloned unilog:
```
$ clojure -J-Dclojure.main.report=stderr -Tnvd nvd.task/check :classpath \""$(lein classpath)\""
```
In the output I see:
```
+-----------------------------+--------------------------------+
| dependency                  | status                         |
+-----------------------------+--------------------------------+
| jackson-databind-2.13.3.jar | CVE-2022-42004, CVE-2022-42003 |
+-----------------------------+--------------------------------+
2 vulnerabilities detected. Severity: HIGH
```

As of this writing unilog is using the latest [net.logstash.logback/logstash-logback-encoder](net.logstash.logback/logstash-logback-encoder) (v7.2) and it seem they haven't bumped to a fixed jackson yet.  

For now, I'll just bring in a newer jackson-databind in cljdoc to address.

**Comment by pyr** [comment]
Hi @lread,

Thanks for this, I just released 0.7.31 to address the issue!

**Comment by lread** [comment]
Thanks @pyr!
